packages/medusa/src/types/subscribers.ts

```diff
@@ -10,7 +10,17 @@ export type SubscriberConfig = {
 }
 
 export type SubscriberArgs<T = unknown> = {
-  data: T
+  data:
```

Comment:
suggestion: you can use `MessageBody<T>` from the types package

Reply:
I'll open another PR for it 👍🏻